src/constants/constant.js

```diff
@@ -0,0 +1,3 @@
+export const CODE_EDITOR_DEFAULT_VALUE = `function yourFunction(a, b) {
+  // your code...
+}`;
```

Comment:
코드에디터의 기본값을 상수로 관리하신 부분은 너무 좋은 것 같아요 💯💯

다만, 가독성을 위해 아래와 같은 방식으로 줄바꿈하는 것은 어떨까요?
```suggestion
export const CODE_EDITOR_DEFAULT_VALUE = `
function yourFunction(a, b) {
  // your code...
}
`;
```